tests/suite_test.go

```diff
@@ -60,6 +62,9 @@ var (
 	//KeyPVBackendStorageClass defines default provisioner to be used
 	// to create the data(export) directory for NFS server
 	KeyPVBackendStorageClass = "BackendStorageClass"
+
+	// backend pvc configuration for integration test
+	integrationTestFinalizer = "it.nfs.openebs.io/test-protection"
```

Comment:
let's extend this comment specifically to the finalizer and why it is required?

Reply:
updated code comments as detailed explanation